scripts/genproto.sh

```diff
@@ -131,8 +131,8 @@ This API reference is autogenerated from the named \`.proto\` files." || exit 2
   log_success -e "\\nTo publish the API references, copy the following files"
   log_success "  - ${API_REFERENCE_FILE}"
   log_success "  - ${API_REFERENCE_CONCURRENCY_FILE}"
-  log_success "to the etcd-io/website repo under the /content/en/docs/next/dev-guide/ folder."
-  log_success "(https://github.com/etcd-io/website/tree/main/content/en/docs/next/dev-guide)"
+  log_success "to the etcd-io/website repo under the /content/en/docs/v3.5/dev-guide/ folder."
+  log_success "(https://github.com/etcd-io/website/tree/main/content/en/docs/v3.5/dev-guide)"
```

Comment:
> In the new model `main` should point on '3.6' directory, such that API changes specific to v3.6 will not pollute the v3.5 docs, right ?

I think for now, we should leave these lines alone, but eventually point them to `v3.6`.

As of right now, neither `next` nor `v3.6` exist in the folder structure of the Docs.